MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -375,7 +389,7 @@ - (void)pushDebugEventWithAttributes:(MMEMapboxEventAttributes *)attributes {
     [combinedAttributes setObject:[self.dateWrapper formattedDateStringForDate:[self.dateWrapper date]] forKey:@"created"];
     [combinedAttributes setObject:self.uniqueIdentifer.rollingInstanceIdentifer forKey:@"instance"];
     MMEEvent *debugEvent = [MMEEvent debugEventWithAttributes:combinedAttributes];
-    [MMEEventLogger logEvent:debugEvent];
+    [MMEEventLogger.sharedLogger logEvent:debugEvent];
```

Comment:
For all of the existing invocations of `pushDebugEventWithAttributes:` we should make sure their attributes have a `MMEDebugEventType` and `MMEDebugEventType{foo}` value that makes sense for the debug log entries they represent.